src/test/clojure/clara/test_accumulators.clj

```diff
@@ -59,10 +59,15 @@
         session (-> empty-session
                     (insert (->Temperature 30 "MCI"))
                     (insert (->Temperature 10 "MCI"))
-                    (insert (->Temperature 80 "MCI")))
+                    (insert (->Temperature 80 "MCI"))
+                    fire-rules)
 
-        min-retracted (retract session (->Temperature 10 "MCI"))
-        max-retracted (retract session (->Temperature 80 "MCI"))]
+        min-retracted (-> session
```

Comment:
You are correct, looks like we don't have an assertion on that currently at https://github.com/cerner/clara-rules/blob/0.13.0/src/test/clojure/clara/test_accumulators.clj#L64  That should be fixed (it can be with this or another quick commit if we don't make these changes)